lib/samples/find_address_with_reverse_geocode/README.metadata.json

```diff
@@ -3,9 +3,9 @@
     "description": "Use an online service to find the address for a tapped point.",
     "ignore": false,
     "images": [],
-    "keywords": [],
+    "keywords": ["address", "geocode", "locate", "reverse geocode", "search"],
     "redirect_from": [],
-    "relevant_apis": [],
+    "relevant_apis": ["GeocodeParameters", "LocatorTask", "ReverseGeocodeParameters"],
     "snippets": [],
```

Comment:
@jenmerritt I followed the common design README to populate this metadata but need to confirm if everything is correct